src/grpc/types/common.rs

```diff
@@ -0,0 +1,130 @@
+//! Helper structs and implementations for [crate::grpc].
+
+use std::convert::TryFrom;
+
+// zeroize Entropy and Password
+use zeroize::Zeroize;
+
+use crate::{
+    grpc::types::{gg20, multisig},
+    kv_manager::{error::KvError, kv::Kv},
+};
+
+pub type MessageDigest = tofn::gg20::sign::MessageDigest;
```

Comment:
MessageDigest will be exposed in tofn through both gg20 and multisig.  I guess in `common.rs` you'll just need to pick one of them.